### PR TITLE
code-server 3.8.1

### PR DESCRIPTION
**Pull request by jsjoeio**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
You didn't update the checksum:
```
==> Verifying checksum for 'd2b831369d4c13402a2965a76293f76b23a578ecfa4e2ec6120ad54a4d1ec3fa--code-server-3.8.1.tgz'
Error: SHA256 mismatch
Expected: a58fef2c00cf1ea77697b0782ef646d9985ad12b93b3542926579d7a54475760
  Actual: 0838c6a844695126a780915eab4e4b7864016bee6fc17f58ab19998573000c9b
```

Are you sure you used `bump-formula-pr`?

File: Formula/code-server.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class CodeServer < Formula
   desc "Access VS Code through the browser"
   homepage "https://github.com/cdr/code-server"
-  url "https://registry.npmjs.org/code-server/-/code-server-3.8.0.tgz"
+  url "https://registry.npmjs.org/code-server/-/code-server-3.8.1.tgz"
   sha256 "a58fef2c00cf1ea77697b0782ef646d9985ad12b93b3542926579d7a54475760"
```
Comment (carlocab) [inline Formula/code-server.rb:5]:
```suggestion
  sha256 "0838c6a844695126a780915eab4e4b7864016bee6fc17f58ab19998573000c9b"
```

**Comment by carlocab** [comment]
I've updated it for you.

**Comment by jsjoeio** [comment]
Yes - here's what I ran
```
brew bump-formula-pr --strict code-server --url=https://registry.npmjs.org/code-server/-/code-server-3.8.1.tgz --sha256=a58fef2c00cf1ea77697b0782ef646d9985ad12b93b3542926579d7a54475760
```

image

Ahhh... I know what I did wrong. I wasn't supposed to use the `--sha256` was I? Sorry, first time doing this!

**Comment by jsjoeio** [comment]
Thank you @carlocab 🙏

**Comment by carlocab** [comment]
>I wasn't supposed to use the `--sha256` was I?

Bingo. Even easier:
```
brew bump-formula-pr --version=3.8.1 code-server
```

**Comment by jsjoeio** [comment]
oh man! i'll have to update our docs for that - i was telling my coworkers how slick `bump-formula-pr` is. Thanks for the tip! 🙌

**Comment by carlocab** [comment]
Yep, if the URL can easily be changed by a `sed`-like replacement, then specifying `--version` on its own should be enough. The rest is needed only if it needs more manual intervention.

**Comment by jsjoeio** [comment]
3.8.1 is live on npm - marking as ready!

image

**Review by carlocab — APPROVED** [review]
Thanks for your first contribution to homebrew/core, @jsjoeio!

**Pull request closed without merging**